pkg/ignore/ignore_test.go

```diff
@@ -38,7 +38,7 @@ func TestIgnore_AddIgnoreFiles(t *testing.T) {
 }
 
 func TestAddRecursiveGitIgnores(t *testing.T) {
-	dir, err := ioutil.TempDir(t.TempDir(), "")
+	dir, err := ioutil.TempDir(os.TempDir(), "")
```

Comment:
I'm ok with this, but one of the benefits of `t.TempDir()` is that the directory is automatically cleaned up at the end of the test (https://golang.org/doc/go1.15#testing, https://golang.org/pkg/testing/#T.TempDir). The only thing I ask is that you add a cleanup, something like `defer os.Remove(dir)`.

Overall I would prefer to maintain few versions of go since this `woke` isn't a library, but I don't feel super strongly about it. Thanks for the contribution!

Reply:
Yeah, moving into 115 seems fine too, but then I'd do that consistently (benchmarks use `os.TempDir` currently). Just lemme know which way you prefer.

And my Fedora currently only has 1.14 based Golang too unfortunately :sweat_smile:

Reply:
In that case, I'm good with making this compatible with 1.14.

The advantage of `t.TempDir()` is that it handles cleanup automatically, but because it's only available in 1.15, I'd say anywhere `os.TempDir()` is used in a test, make sure that the directory is cleaned up at the end of the test.

Reply:
Pushed a version that at least has the same auto-cleanup behavior for tests using `newFile`